pkg/repositories/config/migrations.go

```diff
@@ -199,4 +199,14 @@ var Migrations = []*gormigrate.Migration{
 			return tx.Exec("ALTER TABLE workflows ADD COLUMN IF NOT EXISTS state integer;").Error
 		},
 	},
+	// Modify the executions & node_executison table, if necessary
+	{
+		ID: "2020-04-29-executions",
+		Migrate: func(tx *gorm.DB) error {
+			return tx.Exec("DO we need to do migrations for nullable column adds?").Error
```

Comment:
yep, we still need to add the schema to the table - without this the most recent version of the executions model will determine the table schema

Reply:
so alter table?

Reply:
and how about indexes? This sounds a little weird. Can we just not run auto-migration on the model?